README.md

```diff
@@ -1,3 +1,3 @@
 ## Snyk Tag Monitor
 
-Snyk has a hard limit of 1000 unique key-value pairs that can be used as tags. This scheduled lambda checks the number of tags we are using, and sends an alert if that number is too high.
\ No newline at end of file
+Snyk has a hard limit of 1000 unique key-value pairs that can be used as tags. This scheduled lambda checks the number of tags we are using, and sends an alert if that number is too high.
```

Comment:
Can this README be a bit more verbose about what's happening? How often does the lambda run, what's the limit? How the alert is sent & what to do about it if we get one?

Reply:
Re: what to do about it if we get one

we wanted to keep this PR just about the alerting. Follow-up is going to be a whole separate PR, as it will likely rely on some additional tracking/storage set up and requires a fair amount of context being explained(why do we tag our repos, what is an orphaned tag? how can i tell if it's orphaned? how do i get rid of them?), as well as potentially storing some historical data, now we know that tags are going up way faster than we thought. Maybe a dashboard is the way to go after all...

I'll modify the readme to reflect the rest of the comments though.